homeassistant/components/rainmachine/sensor.py

```diff
@@ -125,6 +151,26 @@ class RainMachineSensorCompletionTimerDescription(
         api_category=DATA_RESTRICTIONS_UNIVERSAL,
         data_key="freezeProtectTemp",
     ),
+    RainMachineSensorDataDescription(
+        key=TYPE_LAST_LEAK_DETECTED,
+        name="Last leak detected",
+        icon="mdi:pipe-leak",
+        entity_category=EntityCategory.DIAGNOSTIC,
+        device_class=SensorDeviceClass.TIMESTAMP,
```

Comment:
Is this only applicable if there's a sensor attached to the controller? If so (and we can assume that's not the norm), I would make it disabled by default (similar to the other flow sensors):

```python
entity_registry_enabled_default=False
```